assessments/2022-12-05-strings/code.js

```diff
@@ -0,0 +1,25 @@
+const upToX = (s) => {
+  return s.substring(0, s.indexOf('x'))
+}
+
+const charactersAround = (s, i) => {
+  return s.substring(i - 1) + s.substring(i + 1);
+}
+const middle = (s) => {
+  return s.substring(0, s.length/2) + s.substring(s.length/2);
+}
+const pair = (s1, s2) => {
+  return s1 + (' ') + ('and') + (' ') + s2;
```

Comment:
This is fine. You could leave out some of the parentheses and write the and with spaces around it as a single string:

```suggestion
  return s1 + ' and ' + s2;
```